railties/lib/rails/generators/rails/plugin/templates/bin/rails.tt

```diff
@@ -1,4 +1,5 @@
-# This command will automatically be run when you run "rails" with Rails 4 gems installed from the root of your application.
+# This command will automatically be run when you run "rails" with Rails 5 gems
```

Comment:
I'm not sure this document should contain a version number anymore. Otherwise it will just keep getting out of date.
I think it's fine to assume that everyone generating new Rails apps will be running `> 4`